### DISCUSSION TITLE
NumPy Core Multi Array failed to import

### DISCUSSION THREAD

**Discussion by soundlab** [mcguirepr89/BirdNET-Pi#1223] [Q&A]
<img width="1378" alt="Screenshot 2024-06-29 at 3 58 04 PM" src="https://github.com/mcguirepr89/BirdNET-Pi/assets/5204902/d0ed1c15-614b-492f-ab55-817172522602">

I'm seeing this error in the log on BirdNet.  I'm able to access the GUI, and see the spectrogram and hear the audio, but no detections have occurred despite testing with a known local sound source.

I'm running a Raspberry Pi 5 with the recommended 64 bit lite and packages sourced from mcguirepr89's Raspberry Pi 5 page.

Any ideas ?

Any help is greatly appreciated!  

Thanks

**Comment by krishfhc** [comment]
I am also having this issue! Cross-posted [here](https://github.com/pddpauw/BirdPi/issues/5).

**Reply by soundlab** [reply]
Thanks @krishfhc!   I will follow up on your detailed thread
